src/index.js

```diff
@@ -10,10 +10,15 @@ import blueprint from './routes/index.js'
 import middlewares from './utils/middlewares/site.js'
 import { koaLogger } from './utils/logging.js'
 import Config from './config.js'
+import SearchEngine from './services/elasticsearch/index.js'
 
 /** @type {import('../types/app').IApp} */
 export const app = new Koa({ proxy: true })
 
+//Search
+await SearchEngine.config()
```

Comment:
this cause tests error I think you need to init searchEngine conditionaly by env to pass this issue

Reply:
> this cause tests error I think you need to init searchEngine conditionaly by env to pass this issue

✔️ I applied your suggestion